### ISSUE TITLE
Dependent Packages problem

### ISSUE THREAD

**Issue by BuddhiLW** [SciML/SciMLTutorials.jl#359]
Calling the solver gives,
"UndefVarError: Rosenbrock23 not defined"

Using the tutorial examples,

![pic-selected-201218-1510-43](https://user-images.githubusercontent.com/59554283/102646765-63eca700-4143-11eb-97bd-21d203d7614c.png)

**Comment by ChrisRackauckas** [comment]
Julia v1.5?

**Comment by BuddhiLW** [comment]
Yes

![pic-selected-201218-1533-22](https://user-images.githubusercontent.com/59554283/102648502-6270ae00-4146-11eb-8e80-f739fcd2ec58.png)

**Comment by ChrisRackauckas** [comment]
What does `]st -m` show?

**Comment by BuddhiLW** [comment]
(@v1.5) pkg> st -m
Status `~/.julia/environments/v1.5/Manifest.toml`
  [c3fe647b] AbstractAlgebra v0.11.2
  [621f4979] AbstractFFTs v0.5.0
  [79e6a3ab] Adapt v2.3.0
  [ec485272] ArnoldiMethod v0.0.4
  [7d9fca2a] Arpack v0.4.0
  [68821587] Arpack_jll v3.5.0+3
  [4fba245c] ArrayInterface v2.14.5
  [4c555306] ArrayLayouts v0.4.11
  [56f22d72] Artifacts v1.3.0
  [13072b0f] AxisAlgorithms v1.0.0
  [aae01518] BandedMatrices v0.15.25
  [e28b5b4c] Bootstrap v2.2.1
  [764a87c0] BoundaryValueDiffEq v2.5.0
  [6e34b625] Bzip2_jll v1.0.6+5
  [fa961155] CEnum v0.4.1
  [336ed68f] CSV v0.7.10
  [83423d85] Cairo_jll v1.16.0+6
  [324d7699] CategoricalArrays v0.8.3
  [d360d2e6] ChainRulesCore v0.9.24
  [aaaa29a8] Clustering v0.14.2
  [da1fd8a2] CodeTracking v1.0.5
  [35d6a980] ColorSchemes v3.10.2
  [3da002f7] ColorTypes v0.10.9
  [5ae59095] Colors v0.12.4
  [861a8166] Combinatorics v1.0.2
  [bbf7d656] CommonSubexpressions v0.3.0
  [34da2185] Compat v3.25.0
  [e66e0078] CompilerSupportLibraries_jll v0.3.4+0
  [8f4d0f93] Conda v1.5.0
  [187b0558] ConstructionBase v1.0.0
  [d38c429a] Contour v0.5.7
  [a8cc5b0e] Crayons v4.0.4
  [717857b8] DSP v0.6.9
  [9a962f9c] DataAPI v1.4.0
  [a93c6f00] DataFrames v0.21.8
  [864edb3b] DataStructures v0.18.8
  [e2d170a0] DataValueInterfaces v1.0.0
  [bcd4f6db] DelayDiffEq v5.27.0
  [2b5f629d] DiffEqBase v6.52.1
  [459566f4] DiffEqCallbacks v2.15.0
  [5a0ffddc] DiffEqFinancial v2.4.0
  [c894b116] DiffEqJump v6.11.0
  [77a26b50] DiffEqNoiseProcess v5.5.0
  [055956cb] DiffEqPhysics v3.7.0
  [163ba53b] DiffResults v1.0.3
  [b552c78f] DiffRules v1.0.2
  [0c46a032] DifferentialEquations v6.15.0
  [c619ae07] DimensionalPlotRecipes v1.2.0
  [b4f34e82] Distances v0.9.2
  [31c24e10] Distributions v0.23.12
  [ffbed154] DocStringExtensions v0.8.3
  [5ae413db] EarCut_jll v2.1.5+1
  [2e619515] Expat_jll v2.2.7+6
  [d4d017d3] ExponentialUtilities v1.8.0
  [e2ba6199] ExprTools v0.1.3
  [8f5d6c58] EzXML v1.1.0
  [c87230d0] FFMPEG v0.4.0
  [b22a6f82] FFMPEG_jll v4.3.1+4
  [7a1cc6ca] FFTW v1.2.4
  [f5851436] FFTW_jll v3.3.9+6
  [9aa1b823] FastClosures v0.3.2
  [1a297f60] FillArrays v0.9.7
  [6a86dc24] FiniteDiff v2.7.2
  [53c48c17] FixedPointNumbers v0.8.4
  [a3f928ae] Fontconfig_jll v2.13.1+14
  [59287772] Formatting v0.4.2
  [f6369f11] ForwardDiff v0.10.14
  [d7e528f0] FreeType2_jll v2.10.1+5
  [559328eb] FriBidi_jll v1.0.5+6
  [069b7b12] FunctionWrappers v1.1.1
  [0656b61e] GLFW_jll v3.3.2+1
  [38e38edf] GLM v1.3.11
  [28b8d3ca] GR v0.53.0
  [d2c73de3] GR_jll v0.53.0+0
  [01680d73] GenericSVD v0.3.0
  [5c1252a2] GeometryBasics v0.3.4
  [78b55507] Gettext_jll v0.20.1+7
  [7746bdde] Glib_jll v2.59.0+4
  [42e2da0e] Grisu v1.0.0
  [cd3eb016] HTTP v0.8.19
  [eafb193a] Highlights v0.4.5
  [0e44f5e4] Hwloc v1.1.0
  [e33a78d0] Hwloc_jll v2.2.0+0
  [09f84164] HypothesisTests v0.10.2
  [7073ff75] IJulia v1.23.1
  [615f187c] IfElse v0.1.0
  [d25df0c9] Inflate v0.1.2
  [83e8ac13] IniFile v0.5.0
  [1d5cc7b8] IntelOpenMP_jll v2018.0.3+0
  [a98d9a8b] Interpolations v0.13.1
  [d8418881] Intervals v1.5.0
  [41ab1584] InvertedIndices v1.0.0
  [c8e1da08] IterTools v1.3.0
  [42fd0dbc] IterativeSolvers v0.8.4
  [82899510] IteratorInterfaceExtensions v1.0.0
  [1019f520] JLFzf v0.1.3
  [692b3bcd] JLLWrappers v1.1.3
  [682c06a0] JSON v0.21.1
  [aacddb02] JpegTurbo_jll v2.0.1+3
  [aa1ae85d] JuliaInterpreter v0.8.7
  [5ab0869b] KernelDensity v0.6.2
  [c1c5ebd0] LAME_jll v3.100.0+3
  [dd4b983a] LZO_jll v2.10.0+3
  [b964fa9f] LaTeXStrings v1.2.0
  [2ee39098] LabelledArrays v1.4.0
  [23fbe1c1] Latexify v0.14.6
  [dd192d2f] LibVPX_jll v1.9.0+1
  [e9f186c6] Libffi_jll v3.2.1+4
  [d4300ac3] Libgcrypt_jll v1.8.5+4
  [7e76a0d4] Libglvnd_jll v1.3.0+3
  [7add5ba3] Libgpg_error_jll v1.36.0+3
  [94ce4f54] Libiconv_jll v1.16.0+7
  [4b2f31a3] Libmount_jll v2.34.0+3
  [89763e89] Libtiff_jll v4.1.0+2
  [38a345b3] Libuuid_jll v2.34.0+7
  [093fc24a] LightGraphs v1.3.4
  [d3d80556] LineSearches v7.1.1
  [4345ca2d] Loess v0.5.3
  [bdcacae8] LoopVectorization v0.9.8
  [6f1432cf] LoweredCodeUtils v1.2.6
  [d00139f3] METIS_jll v5.1.0+5
  [856f044c] MKL_jll v2020.2.254+0
  [1914dd2f] MacroTools v0.5.6
  [739be429] MbedTLS v1.0.3
  [c8ffd9c3] MbedTLS_jll v2.16.8+1
  [442fdcdd] Measures v0.3.1
  [e1d29d7a] Missings v0.4.4
  [78c3b35d] Mocking v0.7.1
  [961ee093] ModelingToolkit v4.2.0
  [46d2c3a1] MuladdMacro v0.2.2
  [f9640e96] MultiScaleArrays v1.8.1
  [6f286f6a] MultivariateStats v0.7.0
  [ffc61752] Mustache v1.0.9
  [d41bc354] NLSolversBase v7.7.1
  [2774e3e8] NLsolve v4.5.0
  [77ba4419] NaNMath v0.3.5
  [b8a86587] NearestNeighbors v0.4.8
  [8913a72c] NonlinearSolve v0.3.1
  [6fe1bfb0] OffsetArrays v1.4.1
  [e7412a2a] Ogg_jll v1.3.4+2
  [5fb14364] OhMyREPL v0.5.10
  [4536629a] OpenBLAS_jll v0.3.9+5
  [458c3c95] OpenSSL_jll v1.1.1+6
  [efe28fd5] OpenSpecFun_jll v0.5.3+4
  [91d4177d] Opus_jll v1.3.1+3
  [bac558e1] OrderedCollections v1.3.2
  [1dea7af3] OrdinaryDiffEq v5.46.0
  [2f80f16e] PCRE_jll v8.42.0+4
  [90014a1f] PDMats v0.10.1
  [65888b18] ParameterizedFunctions v5.7.0
  [d96e819e] Parameters v0.12.1
  [69de0a69] Parsers v1.0.15
  [b98c9c47] Pipe v1.3.0
  [30392449] Pixman_jll v0.40.0+0
  [ccf2f8ad] PlotThemes v2.0.0
  [995b91a9] PlotUtils v1.0.8
  [91a5bcdd] Plots v1.9.1
  [e409e4f3] PoissonRandom v0.4.0
  [f27b6e38] Polynomials v1.2.0
  [2dfb63ee] PooledArrays v0.5.3
  [438e738f] PyCall v1.92.1
  [d330b81b] PyPlot v2.9.0
  [ede63266] Qt_jll v5.15.2+0
  [1fd47b50] QuadGK v2.4.1
  [74087812] Random123 v1.2.0
  [fb686558] RandomExtensions v0.4.3
  [e6cf234a] RandomNumbers v1.4.0
  [c84ed2f1] Ratios v0.4.0
  [3cdcf5f2] RecipesBase v1.1.1
  [01d81517] RecipesPipeline v0.2.1
  [731186ca] RecursiveArrayTools v2.7.2
  [f2c3362d] RecursiveFactorization v0.1.6
  [189a3867] Reexport v0.2.0
  [ae029012] Requires v1.1.2
  [ae5879a3] ResettableStacks v1.1.0
  [295af30f] Revise v3.1.10
  [79098fc4] Rmath v0.6.1
  [f50d1b31] Rmath_jll v0.2.2+1
  [f2b01f46] Roots v1.0.7
  [7e49a35a] RuntimeGeneratedFunctions v0.4.2
  [476501e8] SLEEFPirates v0.6.1
  [1bc83da4] SafeTestsets v0.0.1
  [30cb0354] SciMLTutorials v0.8.0 `https://github.com/SciML/SciMLTutorials.jl#master`
  [6c6a2e73] Scratch v1.0.3
  [91c51154] SentinelArrays v1.2.16
  [efcf1570] Setfield v0.7.0
  [1277b4bf] ShiftedArrays v1.0.0
  [992d4aef] Showoff v0.3.2
  [699a6c99] SimpleTraits v0.9.3
  [b85f4697] SoftGlobalScope v1.1.0
  [a2af1166] SortingAlgorithms v0.3.1
  [47a9eef4] SparseDiffTools v1.10.2
  [276daf66] SpecialFunctions v0.10.3
  [90137ffa] StaticArrays v0.12.5
  [2913bbd2] StatsBase v0.33.2
  [4c63d2b9] StatsFuns v0.9.6
  [2cb19f9e] StatsKit v0.3.0
  [3eaba693] StatsModels v0.6.15
  [9672c7b4] SteadyStateDiffEq v1.5.1
  [789caeaf] StochasticDiffEq v6.27.0
  [09ab397b] StructArrays v0.4.4
  [856f2bd8] StructTypes v1.2.1
  [bea87d4a] SuiteSparse_jll v5.4.0+9
  [c3572dad] Sundials v4.3.0
  [fb77eaff] Sundials_jll v5.2.0+1
  [d1185830] SymbolicUtils v0.6.2
  [3783bdb8] TableTraits v1.0.0
  [bd369af6] Tables v1.2.2
  [9e3dc215] TimeSeries v0.19.1
  [f269a46b] TimeZones v1.5.3
  [a759f4b9] TimerOutputs v0.5.7
  [0796e94c] Tokenize v0.5.8
  [a2a6695c] TreeViews v0.3.0
  [3a884ed6] UnPack v1.0.2
  [1986cc42] Unitful v1.5.0
  [3d5dd08c] VectorizationBase v0.14.2
  [81def892] VersionParsing v1.2.0
  [19fa3120] VertexSafeGraphs v0.1.2
  [a2964d1f] Wayland_jll v1.17.0+4
  [2381bf8a] Wayland_protocols_jll v1.18.0+4
  [44d3d7a6] Weave v0.10.6
  [efce3f68] WoodburyMatrices v0.5.3
  [02c8fc9c] XML2_jll v2.9.10+3
  [aed1982a] XSLT_jll v1.1.33+4
  [4f6342f7] Xorg_libX11_jll v1.6.9+4
  [0c0b7dd1] Xorg_libXau_jll v1.0.9+4
  [935fb764] Xorg_libXcursor_jll v1.2.0+4
  [a3789734] Xorg_libXdmcp_jll v1.1.3+4
  [1082639a] Xorg_libXext_jll v1.3.4+4
  [d091e8ba] Xorg_libXfixes_jll v5.0.3+4
  [a51aa0fd] Xorg_libXi_jll v1.7.10+4
  [d1454406] Xorg_libXinerama_jll v1.1.4+4
  [ec84b674] Xorg_libXrandr_jll v1.5.2+4
  [ea2f1a96] Xorg_libXrender_jll v0.9.10+4
  [14d82f49] Xorg_libpthread_stubs_jll v0.1.0+3
  [c7cfdc94] Xorg_libxcb_jll v1.13.0+3
  [cc61e674] Xorg_libxkbfile_jll v1.1.0+4
  [12413925] Xorg_xcb_util_image_jll v0.4.0+1
  [2def613f] Xorg_xcb_util_jll v0.4.0+1
  [975044d2] Xorg_xcb_util_keysyms_jll v0.4.0+1
  [0d47668e] Xorg_xcb_util_renderutil_jll v0.3.9+1
  [c22f9ab0] Xorg_xcb_util_wm_jll v0.4.1+1
  [35661453] Xorg_xkbcomp_jll v1.4.2+4
  [33bec58e] Xorg_xkeyboard_config_jll v2.27.0+4
  [c5fb5394] Xorg_xtrans_jll v1.4.0+3
  [ddb6d928] YAML v0.4.4
  [c2297ded] ZMQ v1.2.1
  [8f1865be] ZeroMQ_jll v4.3.2+5
  [83775a58] Zlib_jll v1.2.11+18
  [3161d3a3] Zstd_jll v1.4.5+2
  [700de1a5] ZygoteRules v0.2.1
  [214eeab7] fzf_jll v0.24.2+0
  [0ac62f75] libass_jll v0.14.0+4
  [f638f0a6] libfdk_aac_jll v0.1.6+4
  [b53b4c65] libpng_jll v1.6.37+6
  [f27f6e37] libvorbis_jll v1.3.6+6
  [1270edf5] x264_jll v2020.7.14+2
  [dfaa095f] x265_jll v3.0.0+3
  [d8fb68d0] xkbcommon_jll v0.9.1+5
  [2a0f44e3] Base64
  [ade2ca70] Dates
  [8bb1440f] DelimitedFiles
  [8ba89e20] Distributed
  [7b1f6079] FileWatching
  [9fa8497b] Future
  [b77e0a4c] InteractiveUtils
  [76f85450] LibGit2
  [8f399da3] Libdl
  [37e2e46d] LinearAlgebra
  [56ddb016] Logging
  [d6f4376e] Markdown
  [a63ad114] Mmap
  [44cfe95a] Pkg
  [de0858da] Printf
  [3fa0cd96] REPL
  [9a3f8284] Random
  [ea8e919c] SHA
  [9e88b42a] Serialization
  [1a1011a3] SharedArrays
  [6462fe0b] Sockets
  [2f01184e] SparseArrays
  [10745b16] Statistics
  [4607b0f0] SuiteSparse
  [8dfed614] Test
  [cf7118a7] UUIDs
  [4ec0a83e] Unicode

**Comment by BuddhiLW** [comment]
I was able to use ModelingToolKit and OrdinaryDiffEq together to solve the same problem.

I implemented a simple pendulum with friction equation both ways, only had success with ModelingToolKit and OrdinaryDiffEq.

Note that, yesterday, I had solved three problems with DifferentialEquations, which arent currently working, due to this error.

![pic-selected-201218-1539-06](https://user-images.githubusercontent.com/59554283/102649380-ce9fe180-4147-11eb-83f7-e70f9ba3b7ac.png)
![pic-window-201218-1537-25](https://user-images.githubusercontent.com/59554283/102649382-cfd10e80-4147-11eb-8ddd-fdfacb42e8d3.png)
